src/cli.js

```diff
@@ -173,6 +173,7 @@ Options
   --step-data=<all/fail/none>              Keep step data for all steps, only failed steps, or no steps
   --test-server=<url>                      Location of test server (e.g., http://localhost:4444/wd/hub for selenium server)
   --version                                Output the version of Smashtest (-v)
+  --recursive                              Scan the current directory and subdirectories for .smash files
```

Comment:
Looks good. This list is alphabetized, so all you need to do it move this line up.